.github/workflows/test.yml

```diff
@@ -2,33 +2,23 @@ name: Tests
 
 on:
   push:
-    branches: [main]
   pull_request:
-    branches: [main]
+  workflow_dispatch:
```

Comment:
This allows manually triggering test runs from the github UI (or github CLI). Not a frequent occurrence, but very handy to already have ready when needed.